frontend/src/utils/Icons/Icons.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 
+import TheatersIcon from '@material-ui/icons/Theaters';
 import WarningIcon from '@material-ui/icons/Warning';
 import InfoIcon from '@material-ui/icons/Info';
 import ErrorIcon from '@material-ui/icons/Error';
 import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import GradeIcon from '@material-ui/icons/Grade';
+import TurnedInIcon from '@material-ui/icons/TurnedIn';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 
 const getIcon = type =>
   ({
```

Comment:
We should be defining the icon map outside of the function and the function should do something like 
```suggestion
  const getIcon = type => iconMap[type]
```